### PR TITLE
feat: add documentation about remote registries

### PR DESCRIPTION
**Pull request by TerryHowe**
I was going to revive [Deepesha's PR ](https://github.com/oras-project/oras-www/pull/252) on this, but with the merge conflicts and changes required, it was too far off. This PR adds documentation on how to use a remote registry with ORAS.

Closes: https://github.com/oras-project/oras-www/pull/252

### CODE REVIEW

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: versioned_docs/version-1.2/how_to_guides/remote_registries.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+The file will be recreated if you deleted it:
+```bash
+% cat artifact.txt 
+hello world
+% 
+```
+
+## View referrers
```
Comment (FeynmanZhou) [inline versioned_docs/version-1.2/how_to_guides/remote_registries.mdx:87]: Should we describe this section as "view artifact"? If we want to demonstrate "view referrers" with `oras discover`, I would suggest adding a step to attach a referrer file to the pushed artifact/image above this step.

See https://oras.land/docs/concepts/reftypes#listing-referrers

**Review by sajayantony — APPROVED** [review]
LGTM and merging for now to avoid losing the PR :) 
We can update this to first push a referrer if we want.

File: versioned_docs/version-1.2/how_to_guides/remote_registries.mdx (hunk 1 above)
Comment (TerryHowe) [reply versioned_docs/version-1.2/how_to_guides/remote_registries.mdx:87]: I created an issue for this and will follow-up shortly on that.

**Pull request merged**